webdriver/tests/bidi/browsing_context/handle_user_prompt/handle_user_prompt.py

```diff
@@ -110,7 +112,9 @@ async def test_prompt(
         assert result == {"type": "null"}
 
 
-@pytest.mark.parametrize("accept", [True, False])
+@pytest.mark.capabilities({"unhandledPromptBehavior": {'default': 'ignore'}})
+@pytest.mark.parametrize("accept", [False])
+# @pytest.mark.parametrize("accept", [True, False])
```

Comment:
This left in debugging code here. I'm going to fix this with my work on https://bugzilla.mozilla.org/show_bug.cgi?id=1904822

Reply:
Oops. Thank you!